### DISCUSSION TITLE
Issue with SVG Icons not displaying correctly

### DISCUSSION THREAD

**Discussion by axelsegebrecht** [CaiJimmy/hugo-theme-stack#633] [Q&A]
Hi all, I didn't want to file a bug report just yet as it is probably me who is doing something wrong.

When I add new SVGs for social media icons (e.g. Mastodon or a simple email icon) it ends up not looking great.

![img](https://res.cloudinary.com/axel/image/upload/v1655384553/screenshots/Example_Site.png) 

[Icon via Iconmonstr](https://iconmonstr.com/email-12-svg/)

The hope was that it looks more like the ones Jimmy is using and I am probably missing something obvious here.

**Comment by CaiJimmy** [comment]
Use [Tabler Icons](https://tablericons.com/)
